app/Transformers/UserTransformer.php

```diff
@@ -43,6 +43,7 @@ class UserTransformer extends Fractal\TransformerAbstract
         'statistics',
         'unranked_beatmapset_count',
         'user_achievements',
+        'visible_banner',
```

Comment:
`tournament_banner`, `tournament_support_banner` or something instead? `visible_banner` feels way too ambiguous at the `UserTransformer` level